docs/how-to-guides/enqueueing-assets-in-the-editor.md

```diff
@@ -0,0 +1,118 @@
+# Enqueueing assets in the Editor
+
+This guide is designed to be the definitive reference for enqueueing assets (scripts and styles) in the Editor. The approaches outlined here represent the recommended practices but keep in mind that this resource will evolve as WordPress does. Updates are encouraged. 
+
+As of WordPress 6.3, the Post Editor is iframed if all registered blocks have a [`Block API version 3`](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/) or higher and no traditional metaboxes are registered. The Site Editor has always been iframed. This guide assumes you are looking to enqueue assets for the iframed Editor, but refer to the backward compatibility section below for additional considerations. 
```

Comment:
Worth noting that enqueuing assets to work for iframing should also work if the editor is not iframed, just not vice versa (things working without iframe will not necessarily work with the iframe).

Reply:
Thanks @ellatrix. I will add this note to the doc later this week. I have a few more minor tweaks to make.